pkg/operators/oci-handler/oci.go

```diff
@@ -264,6 +266,91 @@ func (o *OciHandlerInstance) init(gadgetCtx operators.GadgetContext) error {
 		return fmt.Errorf("unmarshalling metadata: %w", err)
 	}
 
+	// Extract custom params
+	customParams := viper.Sub("params.custom")
+	if customParams != nil {
+		for k := range viper.GetStringMap("params.custom") {
+			log.Debugf("evaluating custom param %q", k)
+			paramSub := customParams.Sub(k)
+			valuesSub := paramSub.Sub("values")
+			if valuesSub == nil {
+				log.Debugf("custom param %q has no values", k)
+				continue
+			}
+			p := &api.Param{
+				Key:          k,
```

Comment:
The Key field might be different than the key of the map.

Notice `-` != `_` in the trace_exec gadget:
```
params:
  ebpf:
    ignore_failed:
      key: ignore-failed 
      defaultValue: "true"
      description: Ignore failed events
```

Reply:
`key` is not being used, instead the actual object key is, right? I don't think we should consider key as a valid field to avoid this ambiguity.